app/views/comments/index.html.erb

```diff
@@ -159,5 +159,6 @@
   <%== YoutubeTag.script %>
   <%== PodcastTag.script %>
   <%== GistTag.script %>
+  <%== RunkitTag.script %>
```

Comment:
what about this line?

Reply:
Ah sorry yes I meant to leave a comment on the PR about this. If you go to the comments index page and add a new comment in `main` just now you'll see an error in the console:

```
base.js:1961 Uncaught (in promise) ReferenceError: activateRunkitTags is not defined
```

It's because this script isn't imported like it is on the article page. Although unrelated, the error was causing cypress tests to fail, so I went ahead and added it in

Reply:
(`activateRunkitTags` is called from `initializeCommentsPage` expecting it to be globally available, but it isn't imported unless we add this RunkitTag script)